sdk/eventhub/service.projects

```diff
@@ -32,8 +37,4 @@
     <ProjectReference Remove="@(ProjectsToRemove)" />
   </ItemGroup>
 
-  <Target Name="EventHubsOutput" BeforeTargets="Build">
```

Comment:
Why'd you pull this?    I'd really like to keep it in there.

Reply:
It's result's in a spew of output when building `eng/services.proj` that is EH specific

Reply:
Can we keep it and add a clause in there that doesn't trigger unless an `SDKType` is set?   That should resolve the central issue while keeping the information in the more specific context.

Reply:
Great idea.